arkouda/plotting.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np  # type: ignore
+from matplotlib import pyplot as plt  # type: ignore
+
+from arkouda.numeric import log
+
+
+def log10(x):
+    basechange = float(np.log10(np.exp(1)))
```

Comment:
i get it now... this should prob get a docstring... the key thing here is implementing `log10(x)` for a `pdarray`, see the imported `log` from `arkouda.numeric`... maybe we should `put `log10` into `arkouda.numeric`

Reply:
maybe also `typeguard` it...

Reply:
@mhmerrill - I moved `log10` to `arkouda.numeric`. Docstring has been added as well as typegaurd.